src/scripts.js

```diff
@@ -16,9 +19,27 @@ const favoriteRecipesSection = document.getElementById('favoriteRecipesSection')
 
 const weeklyRecipesSection = document.getElementById('weeklyRecipesSection');
 
+let searchBar = document.querySelector('.favorites-view__favorites');
+let mainGlide = document.querySelector('.search-ingredient-glide__images-container');
 
 homeViewButton.addEventListener('click', displayHomeView);
 allRecipesButton.addEventListener('click', displayAllRecipes);
+searchBar.addEventListener('keyup', (e) => {
+  mainGlide.innerHTML = '';
+  const searchTerm = e.target.value;
```

Comment:
I would update this to use `.toLowerCase()` with the variable definitions so you don't have to use it three times below (e.g. `const searchTerm = e.target.value.toLowerCase();`).